twitter-finagle/src/main/scala/com/dwolla/util/async/finagle/ThriftClientConfiguration.scala

```diff
@@ -0,0 +1,11 @@
+package com.dwolla.util.async.finagle
+
+import com.twitter.finagle.service.RetryBudget
+import com.twitter.util.Duration
+
+case class ThriftClientConfiguration(
+                                      requestTimeout: Duration = Duration.fromSeconds(30),
+                                      sessionAcquireTimeout: Duration = Duration.fromSeconds(30),
```

Comment:
_nit_: Let's call this `sessionAcquisitionTimeout`

Reply:
Renamed